### PR TITLE
Documented bridge network width option

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/2367.

@yuyangbj and @wjun can you please review? Thanks!

### CODE REVIEW

File: docs/user_doc/vic_app_dev/network_use_cases.md [hunk 1]
```diff
@@ -181,10 +173,18 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 50ca68165532        tomcat:alpine       "catalina.sh run"   7 minutes ago       Up 5 minutes        192.168.78.159:80->8080/tcp   priceless_tesla
 ```
 
+## Create a Bridge Network with Non-Default Subnet Mask
+
+By default, user-defined bridge networks in VCHs have a subnet mask value of 16. The vSphere administrator can also configure a VCH with a different default subnet mask by using the [`--bridge-network-width`](../vic_vsphere_admin/bridge_network.md#bridge-width) option when they deploy the VCH. 
+
+To override the default subnet mask, you can specify a new value by using the `docker network create --subnet` option.
```
Comment (yuyangbj) [inline docs/user_doc/vic_app_dev/network_use_cases.md]: Should we add move "By default xxxxx" to this paragraph below? And adding a Note for the description 'by default'? If not, I am feeling it is weired, actually default behavior and docker network create are  two methods to change the default subnet.

File: docs/user_doc/vic_vsphere_admin/bridge_network.md [hunk 2]
```diff
@@ -47,7 +47,10 @@ If you do not specify `--bridge-network` or if you specify an invalid port group
...
 
-When you specify a bridge network IP range, you specify the IP range as a CIDR. The smallest subnet that you can specify is /16.
+When you specify a bridge network IP range, you specify the IP range as a CIDR. 
+
+- In releases up to and including vSphere Integrated Containers 1.5.1, the smallest subnet that you can specify is /16.
+- In vSphere Integrated Containers 1.5.2 and later, the default subnet is /16, but you can configure smaller default subnets. If you set a subnet value in the `--bridge-network-range` option that is larger than the default of 16, you must also use the `--bridge-network-width` option to increase the size of the default subnet mask for user-defined bridge networks.
```
Comment (yuyangbj) [inline docs/user_doc/vic_vsphere_admin/bridge_network.md]: actually the customer can also configure a bigger default subnet, so we'd better remove the word "smaller".

File: docs/user_doc/vic_vsphere_admin/bridge_network.md [hunk 3]
```diff
@@ -63,6 +66,28 @@ If the default range of 172.16.0.0.0/12 is in use in your network, specify a new
 
 If you specify an invalid value for `--bridge-network-range`, `vic-machine create` fails with an error.
 
+### Bridge Network Width <a id="bridge-width"></a>
+
+If you need to configure bridge networks to have subnet masks that are smaller than the default of 16 bits, you can set a new default bridge network width. For example, if container application developers use `docker network create` to create multiple new user-defined networks in their VCHs, you might need to configure a smaller subnet mask to avoid overuse of the VLANs that are assigned to those VCHs.
```
Comment (yuyangbj) [inline docs/user_doc/vic_vsphere_admin/bridge_network.md]: Should we add a default behavior? By default ip  range is 172.16.0.0/12 and network allocation width is 16.

File: docs/user_doc/vic_app_dev/network_use_cases.md (hunk 1 above)
Comment (stuclem) [reply docs/user_doc/vic_app_dev/network_use_cases.md]: Done

File: docs/user_doc/vic_vsphere_admin/bridge_network.md (hunk 2 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/bridge_network.md]: Changed "smaller" to "different".

File: docs/user_doc/vic_vsphere_admin/bridge_network.md (hunk 3 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/bridge_network.md]: Done

**Comment by stuclem** [comment]
Thanks @yuyangbj. I made the changes. Is this OK now?

**Comment by stuclem** [comment]
Thanks @yuyangbj

**Pull request merged**